### ISSUE TITLE
Setup continuous integration

### ISSUE THREAD

**Issue by zwaldowski** [bignerdranch/Deferred#35]
- [x] Swift 2.0 Travis CI
  * [x] `swift-2_0` branch protection
  * [x] `swift-2_0` branch require status checks
- [x] Swift 1.2 Travis CI
  * [x] `master` branch protection
  * [x] `master` branch require status checks
- [x] Turn off per-push builds for Travis (PR builds only)

**Comment by zwaldowski** [comment]
Waiting on Travis to :cactus: `master` to complete this task. It should be [this build](https://travis-ci.org/bignerdranch/Deferred/builds/87974056).

**Comment by zwaldowski** [comment]
All done. :+1:
